graphics/tinygl/init.cpp

```diff
@@ -44,7 +44,15 @@ void initSharedState(GLContext *c) {
 void endSharedState(GLContext *c) {
 	GLSharedState *s = &c->shared_state;
 
-	free_texture(c, 0);
```

Comment:
I'm not sure about that. `initSharedState` allocates texture 0. Why would endSharedState free more than texture 0?
Is it to prevent memory leaks where some client code forgot to free its textures? Wouldn't that change make those cases harder to debug?